src/Makefile.in

```diff
@@ -138,7 +138,7 @@ LUALIBS=@LUALIBS@
 
 LIBNOIT_OBJS=noit_check_log_helpers.lo noit_fb.lo bundle.pb-c.lo \
 	noit_check_tools_shared.lo stratcon_ingest.lo noit_metric_rollup.lo \
-	noit_metric_director.lo noit_message_decoder.lo noit_metric.lo \
+	noit_metric_director.lo noit_message_decoder.hlo noit_metric.hlo \
```

Comment:
why wouldn't just build everything with -O2 straight up?

Reply:
We hadn't done that in the past, but we could. Needs team consensus IMO.

Reply:
A good question. This can probably be optimized further. This is merely restoring previous functionality - we used to build just these two entries with `-O5` and we were having issues. Due to other fixes, building with optimizations as we were before should be fine now. I reduced the `-O5` to `-02`, per your earlier suggestion.

Reply:
Defaulting to optimizations in code is fine and convenient, but means you're patching something later if you need to build unoptimized for some reason.

Reply:
We could still default to O2 in higher-level automation, i.e. our own package builds.